source/includes/table-supported-mongodb-versions.rst

```diff
@@ -0,0 +1,10 @@
+.. list-table::
+   :header-rows: 1
+   
+   * - Database 
+     - Supported Versions 
+     - Deployments 
+
+   * - MongoDB 
+     - 4.4 and higher
+     - Self hosted, MongoDB Atlas
```

Comment:
The language we settled on for non-Atlas deployments was "self-managed" ([example](https://www.mongodb.com/docs/manual/reference/configuration-options/)).

I'm also not sure if we should also use "self-managed" instead of "self hosted" to describe the non-MongoDB databases in this [table](https://deploy-preview-211--docs-relational-migrator.netlify.app/getting-started/#source-databases-and-versions), like Oracle, SQL Server, etc.